app/src/full/java/io/homeassistant/companion/android/notifications/NotificationActionReceiver.kt

```diff
@@ -63,8 +64,13 @@ class NotificationActionReceiver : BroadcastReceiver() {
                 Toast.makeText(context, R.string.event_error, Toast.LENGTH_LONG).show()
             }
         }
+        val replyText = if (notificationAction.key == "REPLY")
+            RemoteInput.getResultsFromIntent(intent).getCharSequence(MessagingService.KEY_TEXT_REPLY).toString()
+        else
+            "no reply"
```

Comment:
Rather than having `reply_text` in every action event we should probably only add the `reply_text` if there is an actual reply.  You can probably just add it to the `notificationAction.data` here rather than adding to the signature of `fireEvent`.

Reply:
much better thank you :)